src/Engine/GameObject/Pathfinding.cpp

```diff
@@ -1,26 +1,96 @@
 #include "Pathfinding.hpp"
+#include "Scene\Scene.hpp"
 
-std::vector<char> PathfindingMap::Neighbors(char id) {
-	return edges[id];
+#define MAXPATHNODE 1024
```

Comment:
Will we ever have more than 1024 Nodes in a path?

Reply:
No